stable/redis-ha/values.yaml

```diff
@@ -1,54 +1,86 @@
 ## Configure resource requests and limits
 ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
 ##
-resources:
-  server:
+image: redis
+tag: 4.0.11-stretch
+pullPolicy: IfNotPresent
+## replicas number for each component
+replicas: 3
+
+## Redis specific configuration options
+redis:
+  port: 6379
+  masterGroupName: mymaster
+  config:
+    min-slaves-to-write: 1
+    min-slaves-max-lag: 5   # Value in seconds
+    maxmemory: "0"       # Max memory to use for each redis instance. Default is unlimited.
+    maxmemory-policy: "volatile-lru"  # Max memory policy to use for each redis instance. Default is volatile-lru.
+    # Determines if scheduled RDB backups are created. Default is false.
+    # Please note that local (on-disk) RDBs will still be created when re-syncing with a new slave. The only way to prevent this is to enable diskless replication.
+    save: "900 1"
+    # When enabled, directly sends the RDB over the wire to slaves, without using the disk as intermediate storage. Default is false.
+    repl-diskless-sync: "yes"
+    rdbcompression: "yes"
+    rdbchecksum: "yes"
+
+  resources:
     requests:
+
       memory: 200Mi
       cpu: 100m
     limits:
       memory: 700Mi
-  sentinel:
+
+## Sentinel specific configuration options
+sentinel:
+  port: 26379
+  quorum: 2
+  config:
+    down-after-milliseconds: 10000
+    ## Failover timeout value in milliseconds
+    failover-timeout: 180000
+    parallel-syncs: 5
+
+  resources:
     requests:
       memory: 200Mi
       cpu: 100m
     limits:
       memory: 200Mi
 
-## Node labels, tolerations, and affinities for pod assignment
+securityContext:
+  runAsUser: 1000
+  fsGroup: 1000
```

Comment:
`fsGroup` is not valid in the context of `SecurityContext`, but is valid in the context of `PodSecurityContext`

Reply:
I moved the security context to the pod level so that it is now a valid config